publications.json

```diff
@@ -14864,6 +14864,45 @@
     "description": "The purpose of this document is to provide test cases for LightweightM2M Enabler Release V1.0. The implementation of some features is optional for the Clients and/or the Servers in the LightweightM2M Enabler. The tests associated with these optional features are marked as (Includes Optional Features) in the test specification",
     "contentMediaType": "application/pdf"
   },
+  {
+    "organization": "https://technical.openmobilealliance.org/organization-info.json",
+    "enablerName": "OMA Enabler Test Specification for LightweightM2M (LwM2M)",
+    "enablerAbbreviation": "LightweightM2M",
+    "status": "Approved",
+    "version": "V1.0.2",
+    "releaseDate": "2018-08-15",
+    "fileName": "OMA-ETS-LightweightM2M-V1_0_2-20180815-A",
+    "docType": "Enabler Test Specification",
+    "url": "https://openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M-V1_0_2-20180815-A.zip",
+    "description": "This document describes in detail available test cases for LightweightM2M as specified in OMA-TS-LightweightM2M-V1_0. The test cases are split in two categories, conformance and interoperability test cases. The conformance test cases are aimed to verify the adherence to normative requirements described in the technical specifications. The interoperability test cases are aimed to verify that implementations of the specifications work satisfactory. If either conformance or interoperability tests do not exists at the creation of the test specification this part should be marked not available.",
```

Comment:
@gocadimic This description is better to take it from the Introduction. We don't do Conformance test cases. Therefore, this description doesn't provide any input. The content in the Introduction session is better and provides further information.